scripts/sct_maths.py

```diff
@@ -172,14 +177,15 @@ def get_parser():
     filtering = parser.add_argument_group("FILTERING METHODS")
     filtering.add_argument(
         "-smooth",
-        metavar='',
+        metavar=Metavar.list,
+        type=list_type(',', float),
         help='Gaussian smoothing filter with specified standard deviations in mm for each axis (Example: 2,2,1) or '
              'single value for all axis (Example: 2).',
         required=False)
     filtering.add_argument(
         '-laplacian',
-        nargs="+",
```

Comment:
We can't use `nargs="+"` because that only works once in a CLI?

Reply:
The reason I switched here is because `-laplacian` used to be comma-delimited, but `nargs="+"` is space-delimited. I wanted to put things back to what they used to be so that they would stay consistent.

e.g. see this usage which is still buried in the API:

https://github.com/neuropoly/spinalcordtoolbox/blob/69287b0953fad48bbabc4722d74ccf484ebfbeee/spinalcordtoolbox/registration/register.py#L192-L198

In fact... now that I think about it, this call would crash with `nargs="+"` right now.

Reply:
It's possible the other `nargs="+"` calls could be changed too (e.g. `-add`, `-mul`, etc.) to keep syntax consistent.